### ISSUE TITLE
Multiple Actions on Tap

### ISSUE THREAD

**Issue by maniscalcojohn** [custom-cards/button-card#492]
Is it possible to have multiple actions on a tap? I'd like to be able to navigate and switch an input_select on a button press.

**Comment by RomRider** [comment]
You should find a way to do this in this comment:
https://github.com/custom-cards/button-card/issues/314#issuecomment-616258963

**Comment by maniscalcojohn** [comment]
Will that work to navigate? I'm trying to call a service and navigate on a single button

**Comment by RomRider** [comment]
Yes, use navigate as the action and call your service like in the example linked in any other yaml key.

**Comment by borpin** [comment]
Whilst that works, could the `tap_action` be an object list rather than an object. I'm wanting a button to toggle multiple automations at once. Yes I know I can do it another way, but being able to just define multiple entities inside a button would be great. 😄

**Comment by dcapslock** [comment]
`tap_action` is modelled from core. To be different there is (a) code complexity (b) risk of users thinking that core HA supports what button card does (e.g. users try and template javascript `[[[]]]` on core HA cards due to copying button-card examples. If core supports an action list, then button-card could be updated to match. If this is desired then you may wish to read/upvote https://github.com/orgs/home-assistant/discussions/724

**Comment by Mariusthvdb** [comment]
seems the best way to get what you want is running a script with those actions listed.

in that way, tapping the button would be firing an action block of an automation, which in fact is the exact same thing

**Comment by dcapslock** [comment]
Agree. Though seeing a lot of chatter about how complex it is to say navigate but then also call a backend service. This is pretty much impossible except with a Browser Mod sequence as a Browser call (fire-dom-event)

**Comment by Mariusthvdb** [comment]
I see, my response above was no considering the different types of frontend/backend.. sorry for that.

this might be the perfect reason to use BM in that case, I cant imagine the core team even considering the Discussion there.

otoh, one  might be surprised ;-)

suggestion:

add the navigation service to the example in 
https://github.com/custom-cards/button-card/issues/314#issuecomment-616258963 so users can c&p that from within the documentation here
